src/components/SRNEntry.vue

```diff
@@ -104,72 +102,75 @@ export default {
     isAnyUserIDPresent() {
       return (
         this.userIDList != undefined &&
         this.userIdListLength > 0 &&
         this.userIDList[0]["userID"] != ""
       );
     },
-    //checks if multiple entries are allowed and if one entry is already inputted
+    /** Whether multiple entries have been made by the user */
     ifUserEnteredMoreThanOne() {
       return !this.isSingleEntryOnly && this.userIdListLength > 1;
     },
-    /** for now, plio does not support multiple input entries */
+    /** For now, plio does not support multiple input entries */
     isSingleEntryOnly() {
       return this.redirectTo == "plio";
     },
-    /* returns submit button as disabled if any of the following is true:
-     - no SRN has been typed
-     - input is invalid
-     - SRN hasn't been completely typed */
+    /**
+     * Whether the submit button is disabled
+     * true if any of the following conditions are met:
+     * - no SRN has been typed
+     * - input is invalid
+     * - SRN hasn't been completely typed
+     */
     isSubmitButtonDisabled() {
       return (
         !this.isAnyUserIDPresent ||
         this.invalidInputMessage != "" ||
         this.isCurrentEntryIncomplete
       );
     },
-    /* checks if + button should be displayed. Will be activated only if:
-     - multiple entries are allowed
-     - if current input entry is complete
-     - if cap of maximum entries hasn't been reached yet */
+    /**
+     * Checks if + button should be displayed. Will be activated only if:
+     * - multiple entries are allowed
+     * - if current input entry is complete
+     * - if cap of maximum entries hasn't been reached yet
+     */
     isAddButtonAllowed() {
       return (
         !this.isSingleEntryOnly &&
         !this.isCurrentEntryIncomplete &&
         this.userIdListLength < numberOfSRNsAllowed
       );
     },
-    // checks if the current input entry has the required number of characters.
+    /** Checks if the current input entry has the required number of characters */
     isCurrentEntryIncomplete() {
-      return this.getLatestEntry["userID"].length < this.maxLengthOfSRN;
+      return this.latestEntry["userID"].length < this.maxLengthOfSRN;
     },
-    // returns the most recent typed input entry
-    getLatestEntry() {
+    /** Returns the most recently entered input */
+    latestEntry() {
       return this.userIDList.slice(-1)[0];
     },
   },
   methods: {
-    /* determines how the input box should look like depending on which box it is. 
-      - if the input box is the current one (i.e. being typed) and if the input is invalid (invalid input message is being displayed),
-        then, the box has a border of red
-      - if the input box is a past one (i.e. already typed), then the box is shown as unclickable 
-      @param {Number} - index - index of the input box */
+    /** Determines how the input box should look like depending on which input field it is.
```

Comment:
```suggestion
    /** Determines how the input box should look.
```
Only this is enough no?